lib/testUtils/__tests__/createRuleTester.test.js

```diff
@@ -0,0 +1,45 @@
+"use strict"
+
+const createRuleTester = require("../createRuleTester")
+
+describe("createRuleTester", () => {
+  it("is possible to create a tester", () => {
```

Comment:
Great idea to add tests like this! My one reservation is that the one test covers several different possibilities. It *was* possible to create a tester before, right? — just not with `preceedingPlugins`? Maybe that means there should be a couple of separate tests?